www/src/pages/en/introduction.md

```diff
@@ -31,9 +31,9 @@ The _"T3 Stack"_ is a web development stack made by [Theo](https://twitter.com/t
 
 ## What is this? Some kinda template?
 
-Kind of. We love all of the technologies that `create-t3-app` includes. Check out [init.tips](https://init.tips/others) for even more info on topics such as state management and deployment. But we do not believe these are needed on every project.
+Kind of. We love the technologies that the T3 Stack includes but we do not believe that all of them are needed in every project. So we made `create-t3-app` to do one thing: _**Simplify complex boilerplate around the core T3 Stack tech without compromising the pieces modularity.**_
 
-So we made `create-t3-app` to do one thing: _**Simplify complex boilerplate around the core T3 Stack tech without compromising the pieces modularity.**_ This is NOT an all-inclusive template and we expect you to bring your own libraries as well.
+This is NOT an all-inclusive template and we expect you to bring your own libraries as well. Check out [init.tips](https://init.tips/others) for even more info on topics such as state management and deployment.
```

Comment:
I wonder if we should put the "rest" in here and not referring them to init.tips? That site isn't very "maintained"?

Reply:
Yes I think we shouldn't mention init.tips but should mention our docs because they have more info.

Reply:
init.tips is currently being linked in both the readme and existing docs, have removed those links for now. as soon as the docs are out of beta, lets put a link to the docs top and center in the readme instead :)

Reply:
Should we have a page on the docs with further libs where we recommend zustand and others?

I dont have time right now but if you feel like it's a good idea feel free to add